package/yast2-network.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jul 19 08:10:14 UTC 2019 - Josef Reidinger <jreidinger@suse.com>
+
+- avoid dependency on yast2-installation
```

Comment:
A bug number will probably be needed

Reply:
also it is not dependency on yast2-installation

Reply:
well, it is not accepted to target project, it should work without it

Reply:
OK, so then fine with me